command/helpers_test.go

```diff
@@ -685,3 +685,19 @@ func Test_extractVarFlags(t *testing.T) {
 		}, result)
 	})
 }
+
+func Test_extractJobSpecEnvVars(t *testing.T) {
+	ci.Parallel(t)
+
+	t.Run("nil", func(t *testing.T) {
+		must.MapEmpty(t, extractJobSpecEnvVars(nil))
+	})
+
+	t.Run("complete", func(t *testing.T) {
+		result := extractJobSpecEnvVars([]string{"NOMAD_VAR_count=13", "GOPATH=/Users/jrasell/go", "NOMAD_VAR_image=redis:7"})
```

Comment:
nit: I find it easier to read these with one var per line

```suggestion
		result := extractJobSpecEnvVars([]string{
			"NOMAD_VAR_count=13",
			"GOPATH=/Users/jrasell/go",
			"NOMAD_VAR_image=redis:7",
		})
```